frontend/src/components/UserSettings/UserPassword.vue

```diff
@@ -103,7 +103,7 @@ export default {
     disabled() {
       return !(
         this.form.newPassword.password !== '' &&
-        this.form.newPassword.password === this.form.newPassword.passwordRepeat
+        this.form.newPassword.password !== this.form.newPassword.passwordRepeat
       )
     },
```

Comment:
The enabling behavior of the submit button does not work as expected:
Expected: after the new password has been enteredand been validated, the button should be enabled.

Current behavior:
When entering the new password the first time, the button gets enabled.
When entering the new password the second time,the button gets disabled.